contrib/README.md

```diff
@@ -0,0 +1,368 @@
+# Event Gateway on Kubernetes 
+
+This chart deploys the Event Gateway with etcd onto a Kubernetes cluster. Please note, the default instructions expect
+an existing kubernetes cluster that supports ingress, such as GKE. If your environment doesn't have ingress support
+set up, please follow the [minikube](MINIKUBE.md) instructions to set this up for your development environment.  
+
+## Contents
+
+1. [Quickstart](#quickstart)
+    1. [Using helm](#using-helm)
+    1. [Using custom resources](#using-custom-resources)
+1. [Examples](#examples)
```

Comment:
This section should ideally take the user through an end to end EG example that it links out to. Just providing Curl examples for the API is not incredibly satisfying as an outcome for what EG enables developers to do. Can be done in a follow up PR. @alexdebrie @sebito91 Please review.